frontend/src/scenes/apps/appMetricsSceneLogic.ts

```diff
@@ -57,6 +83,12 @@ export const appMetricsSceneLogic = kea<appMetricsSceneLogicType>([
     actions({
         setActiveTab: (tab: AppMetricsTab) => ({ tab }),
         setDateFrom: (dateFrom: string) => ({ dateFrom }),
+        openErrorDetailsDrawer: (errorType: string, category: string, jobId?: string) => ({
```

Comment:
Instead of this being a drawer / pop-up window could we use row expansion in the errors table instead (similar to how live events or async migrations pages have multiple tabs there within row expansion)

Reply:
Can you go into why you prefer one over the other here?

Reply:
visually more appealing, easier to close, more consistent with the rest of our app. You can bring this up during the feedback session to see what others think.